### ISSUE TITLE
Trivial 'exit' issue with Minimig OSD when inside MT32-pi menu

### ISSUE THREAD

**Issue by movisman** [MiSTer-devel/Main_MiSTer#362]
Hi,

Just noticed a trivial issue with the Minimig OSD when you are inside the MT32-pi menu.

If you open up the OSD while the core is loaded and select MT32-pi - when that menu loads, if you select the 'exit' button at the bottom, the whole OSD closes. In all other submenus, it goes back one step.

If you press escape while in the MT32-pi submenu, it will go return to the main OSD correctly (it doesn't close the entire OSD).

It is only when selecting 'exit' from this specific menu does this occur. The other submenus are fine.

Thanks!

**Comment by sorgelig** [comment]
yeah, exit option maybe inconsistent.. It supposed to close whole OSD instead going back. Hence the name "exit" not "back".
I will normalize it

**Comment by movisman** [comment]
Hi there,

Thanks for the reply. So, other cores (I tried TG16, Genesis and ZX Spectrum as examples) have the exit label on the main OSD, which closes it. Which is fine. In all submenus, 'exit' is replaced by 'back' which goes up a level to the main OSD without closing it. Again, this is perfect.

Minimig is a bit different, and since you mentioned the above I noticed something else.

So, in all submenus on the Minimig core, there is no 'back' label - all submenus have 'exit' at the bottom instead - even if the current behaviour is set to go back for all of them except the MT32-pi menu.

What I think needs to happen is in all submenus on Minimig, 'exit' should be replaced with 'back', so it is consistent with the other cores. However in the MT32-pi submenu, the actual behaviour should be 'back' as well as being relabelled (it shouldn't close the OSD entirely).

The behaviour on the main 'top level' OSD of the core is fine (exit at bottom, closes OSD when selected).

In Minimig, it is just the submenus which are not quite right. With the exception of /MT32-pi they do all 'behave' correctly (they go back), but they are all incorrectly labelled as 'exit' instead of 'back'. However when inside the MT32-pi submenu, it is both labelled incorrectly as 'exit' and it closes the OSD completely when selected. It should be labelled 'back' and go back to the top level.

Hope that helps to explain! I thought you would want it reporting so it can be tidied up at some point for consistency.

Thanks very much.
